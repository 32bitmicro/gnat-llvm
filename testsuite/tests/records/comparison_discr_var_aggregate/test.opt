ALL XFAIL Support for records with discriminants is incomplete
