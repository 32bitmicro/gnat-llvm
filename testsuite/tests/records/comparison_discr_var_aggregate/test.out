Foo(
    A => 1, B => 2, C => 3,
  --> 123
    D => 1, E => 2, C => 3,
  --> 123
    Equality => False)
  = False
Foo(
    A => 1, B => 2, C => 3,
  --> 123
    D => 1, E => 2, C => 3,
  --> 123
    Equality => True)
  = True

Foo(
    A => 1, B => 2, C => 3,
  --> 123
    D => 4, E => 2, C => 3,
  --> 423
    Equality => False)
  = True
Foo(
    A => 1, B => 2, C => 3,
  --> 123
    D => 4, E => 2, C => 3,
  --> 423
    Equality => True)
  = False

Foo(
    A => 1, B => 2, C => 3,
  --> 123
    D => 1, E => 4, C => 3,
  --> 143
    Equality => False)
  = True
Foo(
    A => 1, B => 2, C => 3,
  --> 123
    D => 1, E => 4, C => 3,
  --> 143
    Equality => True)
  = False

Foo(
    A => 1, B => 2, C => 3,
  --> 123
    D => 1, E => 2, C => 4,
  --> 124
    Equality => False)
  = True
Foo(
    A => 1, B => 2, C => 3,
  --> 123
    D => 1, E => 2, C => 4,
  --> 124
    Equality => True)
  = False

